File: python/banking_system.py
balance=0
print("\nOPTIONS:\n1.Check balance\n2.Deposit money\n3.Withdraw money\n4.Exit")
choice = input("\n :").lower()
while choice!="4":
    amount=0
    if choice=="1" or choice=="check balance":
        print("Balance: %.2f"%balance)
        choice = input("\n :").lower()
    elif choice=="2" or choice=="deposit":
        amount=float(input("Enter amount to deposit: "))
        balance+=amount
        print("balance: %.2f"%balance)
        choice = input("\n :").lower()
    elif choice=="3" or choice=="withdraw":
        amount=float(input("Enter amount to deposit: "))
        if amount>balance:
            print("Insufficient balance\nAvailable balance: %.2f"%balance)
        else:
            balance-=amount
            print("balance: %.2f"%balance)
            choice = input("\n :").lower()
    else:
        print("Invalid option")
        choice = input("\n :").lower()

File: banking_system.c
//This is the 2nd question of 4th ILBP practical

#include <stdio.h>

int main(void){
	int option;
	float balance=0;
	printf("\nOPTIONS:\n1.check balance\n2.deposit money\n3.withdraw money\n4.exit");
	printf("\nEnter option(Enter index no.): ");
	scanf("%d",&option);
	while (option!=4){	
		float amount=0;
		switch (option){
		case 1:
			printf("\nYou have selected 'check balance'");
			printf("\nYour balance: %.2f",balance);
			printf("\nEnter option(Enter index no.): ");
			scanf("%d",&option);
			break;
		case 2:
			printf("\nYou have selected 'deposit money'\nEnter amount to deposit: ");
			scanf("%f",&amount);
			balance+=amount;
			printf("\nYour balance: %.2f",balance);
			printf("\nEnter option(Enter index no.): ");
			scanf("%d",&option);
			break;
		case 3:
			printf("\nYou have selected 'withdraw money'\nEnter amount to withdraw: ");
			scanf("%f",&amount);
			if (balance - amount < 0){
				printf("\nInsufficient amount");
			}
			else{
			balance-=amount;
			printf("\nYour balance: %.2f",balance);
			}
			printf("\nEnter option(Enter index no.): ");
			scanf("%d",&option);
			break;
		default:
			printf("\nInvalid option");
			printf("\nEnter option(Enter index no.): ");
			scanf("%d",&option);
		}
	}
	printf("\nYour final balance: %.2f\n",balance);
}
